Library/Formula/boost.rb

```diff
@@ -46,6 +48,13 @@ def install
       EOS
     end
 
+    if build.with? "brewed-bzip2"
+      depends_on "bzip2" => :optional
```

Comment:
This line needs to remain just beneath the ` option ` lines up top. It also needs to be fully qualified = ` depends_on "homebrew/dupes/bzip2" => :optional `.